resources/texts/_contentManager.html

```diff
@@ -367,6 +373,12 @@
             <button v-else-if="book.path" v-on:click="openBook(book)">Open</button>
             <button v-else v-on:click="downloadBook(book)">Download</button>
           </span>
+          <span v-if="downloads[book.id] && downloads[book.id].status == 'active'" class="tablecell cell5">
+            <button v-on:click="pauseBook(book)">Pause</button>
+          </span>
+          <span v-if="downloads[book.id] && downloads[book.id].status == 'paused'" class="tablecell cell5">
+                <button v-on:click="resumeBook(book)">Resume</button>
+              </span>
```

Comment:
We should add those button **inside** the already created `span` (and even better, inside the `<template v-if="book.downloadId">`) .

Reply:
![image](https://user-images.githubusercontent.com/47149116/57137545-73717d80-6db0-11e9-812a-60b1ef856480.png)


The button moves according to the window's width, I think that's it's better that this kind of button have a fixed position.

![image](https://user-images.githubusercontent.com/47149116/57137741-0e6a5780-6db1-11e9-9e76-d2be772d0510.png)

Why don't create another span specific to all buttons ?